stonesoup/types/state.py

```diff
@@ -140,14 +142,33 @@ def mean(self):
         """The state mean, equivalent to state vector"""
         return self.state_vector
 
+    @property
+    def state(self):
+        """The state mean, equivalent to state vector"""
+        return self.state_vector
+
```

Comment:
Don't think this should be here, and I don't think it's used (tests passed after I removed it).